### PR TITLE
This should be a requirement for our demo

### PR DESCRIPTION
**Pull request by RuiAAPeres**
This requirement was missing.

### CODE REVIEW

File: Interview/demo.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ The following requirements should be met:
 
 * Use Swift 4.0 or above.
 * The information (posts and post details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
+* Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
```
Comment (sergdort) [inline Interview/demo.md]: Should we highlight it in **bold**? =)

**Comment by RuiAAPeres** [comment]
@sergdort change mande

File: Interview/demo.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ The following requirements should be met:
 
 * Use Swift 4.0 or above.
 * The information (posts and post details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
+* **Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).**
```
Comment (inamiy) [inline Interview/demo.md]:
```suggestion
* **Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).**
* **No retain cycle.**
```

Comment (RuiAAPeres) [reply Interview/demo.md]: Can we just make it plural? 😛 `No retain cycles.`

**Pull request merged**